pyproject.toml

```diff
@@ -33,6 +33,10 @@ uwsgi = "^2.0.23"
 [tool.poetry.group.dev.dependencies]
 isort = "^5.12.0"
 
+
+[tool.poetry.group.linting.dependencies]
+flake8 = "^6.1.0"
```

Comment:
Why not just have this under the `dev` dependencies group?

Reply:
It is now under the `dev` group